src/fedora/24/Dockerfile

```diff
@@ -28,6 +29,57 @@ RUN dnf install -y \
     && npm install -g azure-cli \
     && npm cache clean
 
+# Build and install liblldb development files
```

Comment:
Are you familiar with [multi-stage Dockerfiles](https://docs.docker.com/develop/develop-images/multistage-build/)?  I can't help but wonder if this wouldn't be a cleaner approach for implementing this.  Docker has some pre-release functionality that can parallelize builds of multi-stage dockerfiles.  This would certainly be a benefit here.  This comment has not effect on the resulting image.

Reply:
I hope you won't insist that I do this. I really have spent way too much time on these already.

Reply:
I didn't intend for a change now but rather something to consider down the road.